web/gui/main.js

```diff
@@ -4807,11 +4802,7 @@ function signInDidClick(e) {
 }
 
 function shouldShowSignInBanner() {
```

Comment:
Why not just remove the code where this function is called?

Reply:
Simply because that file will most likely not be maintained :)
I could remove much more cloud-related code, one deletion after another, but it doesn't seem to be a priority so I just went with easiest and safest things to do.